### PR TITLE
To Do List - Interactive list

### PR DESCRIPTION
**Pull request by LarryIVC**
###  **In this PR, I did the following:**

- [ ]  Create a new `branch`.
- [ ]  Refactor the `CSS `to display interactions.
- [ ]  Add `JavaScript `files for new functionality.
- [ ]  Implement a function that `clears all tasks `selected.
- [ ] Manage `JavaScript `events.
- [ ]  All changes to the To Do List are saved in `local storage`.

### CODE REVIEW

File: src/logic.js [hunk 1]
```diff
@@ -1,28 +1,30 @@
+/* eslint-disable */
```
Comment (codepantha) [inline src/logic.js]: - [ ] Please don't disable linters. They are helpful in ensuring that your code adheres to best practices and coding conventions👍

File: src/ClearAll.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
```
Comment (codepantha) [inline src/ClearAll.js]: - [ ] Please don't disable linters. They are helpful in ensuring that your code adheres to best practices and coding conventions👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @LarryIVC 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic.js (hunk 1 above)
Comment (LarryIVC) [reply src/logic.js]: @codepantha, precisely it cannot solve that is why I sent you the following message:

> In order to implement the functionality of the clear all button, I had to make a cyclic call in order to refresh the view and show the changes.

If you could help me with an idea of how to solve it I would greatly appreciate it.

**Comment by BenMukebo** [comment]
Hello @LarryIVC, I'm getting this files error !! could you provide me the screenshort of your app, since you don't have much time !!!!

image

**Comment by BenMukebo** [comment]
@LarryIVC, Your app Still does not work correctly, here is the error message I'm getting while create books:

image

**Comment by LarryIVC** [comment]
@BenMukebo 
image

**Comment by BenMukebo** [comment]
Refresh your page you will see!!!!

File: webpack.config.js [hunk 3]
```diff
@@ -2,10 +2,16 @@ const path = require('path');
...
-
+  mode: 'development',
   entry: {
     index: './src/index.js',
+    logic: './src/logic.js',
+    ClearAll: './scr/ClearAll.js',
+    TaskManager: './scr/TaskManager.js',
+    link: './scr/link.js',
+    Task: './scr/Task.js',
   },
```
Comment (BenMukebo) [inline webpack.config.js]: - [ ] While I appreciate the effort and time you have put into this but there is something you need to work on for your project to be completed,  the disabled linters liners have been removed but I can't create any books instead I noticed that it is throwing an error 👇🏻 


image

https://www.loom.com/share/9177ee1cd0294f749f33e59632f61579

**Comment by LarryIVC** [comment]
@BenMukebo 
https://www.loom.com/share/42df4b0a2c374370896ea7cd68deb08a

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @LarryIVC, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack.config.js (hunk 3 above)
Comment (BenMukebo) [reply webpack.config.js]: @LarryIVC show me all the step starting by creating a book !! If you can provide this, I will approve you Pull Request!!

Comment (BenMukebo) [reply webpack.config.js]: @LarryIVC here is the requiement. 👇🏻 

image

Comment (BenMukebo) [reply webpack.config.js]: I will mark this PR as invalid to give you a change to fix all the create and remove book s issues

**Review by BenMukebo — COMMENTED** [review]
## Status: Invalid ❌

### Hi @LarryIVC,
Please fix the issue mentioned below and then request for code review

### Reason

- [ ] I'm making this PR invalid so it won't use your request count, but you need to fix all issues with creating and deleting books.

#### Please make sure to submit code review when you are done with the changes that was requested

**Don't worry you still have same number of requests left as previous.**


Cheers and Happy Coding :clap: :clap: :clap: 

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: webpack.config.js (hunk 3 above)
Comment (LarryIVC) [reply webpack.config.js]: @BenMukebo, here is the video. 🙏 

https://www.loom.com/share/5247980fedb4430bb2811093e9d2077e

**Comment by LarryIVC** [comment]
@BenMukebo, ok I understand, thanks

File: webpack.config.js (hunk 3 above)
Comment (BenMukebo) [reply webpack.config.js]: @LarryIVC, I need you to show me how you are creating a book 😄

**Comment by LarryIVC** [comment]
@BenMukebo, I make a new video:

**To Do List - Interactive list by LarryIVC - web pack - Watch Video**

[](https://www.loom.com/share/ef526eaa14c844b8b66d81716df76d50)

Sorry for the delay, but I fell asleep.

**Review by peterrobert — APPROVED** [review]
Hi @LarryIVC , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**